picard/metadata.py

```diff
@@ -59,15 +61,30 @@ class Metadata(dict):
 
     multi_valued_joiner = MULTI_VALUED_JOINER
 
-    def __init__(self):
-        super().__init__()
-        self.images = ImageList()
-        self.has_common_images = True
+    def __init__(self, *args, deleted_tags=None, images=None, length=None, **kwargs):
+        self._store = dict()
         self.deleted_tags = set()
         self.length = 0
+        self.images = ImageList()
+        self.has_common_images = True
```

Comment:
`has_common_images` isn't considered in `update()`, not sure about this.
Not even sure what it does.
@phw we'll have to take care of this, after this PR is merged.